### ISSUE TITLE
Cannot build for arch=armv7hf

### ISSUE THREAD

**Issue by RobertUlbrich** [lasote/conan-openssl#46]
https://github.com/lasote/conan-openssl/blob/4baaff48e2efaad6460ad53bc64fc78c5f0cc231/conanfile.py#L142

Conan allows building for armv7hf. Why this restriction?

When arch=armv7hf, OpenSSL build fails:

```
14:43:33 [Linux] OpenSSL/1.0.2m@conan/stable: Building your package in /home/builder/.conan/data/OpenSSL/1.0.2m/conan/stable/build/af4a75b89d714480e33c794b32aea5be5ffa67ce
14:43:33 [Linux] OpenSSL/1.0.2m@conan/stable: Copying sources to build folder
14:43:33 [Linux] OpenSSL/1.0.2m@conan/stable: Generator txt created conanbuildinfo.txt
14:43:33 [Linux] OpenSSL/1.0.2m@conan/stable: Calling build()
14:43:33 [Linux] OpenSSL/1.0.2m@conan/stable: =====> Options:  --with-zlib-include="/home/builder/.conan/data/zlib/1.2.11/conan/stable/package/05c6eb355eb437000a268f4cf2928f212fbf44db/include" --with-zlib-lib="/home/builder/.conan/data/zlib/1.2.11/conan/stable/package/05c6eb355eb437000a268f4cf2928f212fbf44db/lib"
14:43:33 [Linux] OpenSSL/1.0.2m@conan/stable: 
14:43:33 [Linux] OpenSSL/1.0.2m@conan/stable: ERROR: Package 'af4a75b89d714480e33c794b32aea5be5ffa67ce' build failed
14:43:33 [Linux] OpenSSL/1.0.2m@conan/stable: WARN: Build folder /home/builder/.conan/data/OpenSSL/1.0.2m/conan/stable/build/af4a75b89d714480e33c794b32aea5be5ffa67ce
14:43:33 [Linux] ERROR: OpenSSL/1.0.2m@conan/stable: Error in build() method, line 102
14:43:33 [Linux] 	self.unix_build(config_options_string)
14:43:33 [Linux] while calling 'unix_build', line 170
14:43:33 [Linux] 	config_line = "./Configure %s -fPIC %s %s" % (config_options_string, target, extra_flags)
14:43:33 [Linux] 	UnboundLocalError: local variable 'target' referenced before assignment
  
```

**Comment by lasote** [comment]
Have you tried with 1.0.2n?

**Comment by lasote** [comment]
I released also 1.1.0g supporting arm
